runtime/engine/src/test/java/io/aklivity/zilla/runtime/engine/test/internal/expression/TestExpressionResolverSpi.java

```diff
@@ -0,0 +1,20 @@
+package io.aklivity.zilla.runtime.engine.test.internal.expression;
+
+import io.aklivity.zilla.runtime.engine.expression.ExpressionResolverSpi;
+
+public class TestExpressionResolverSpi implements ExpressionResolverSpi
+{
+
+    @Override
+    public String name()
+    {
+        return "test";
+    }
+
+    @Override
+    public String resolve(
+        String var)
+    {
+        return "PASSWORD".equals(var) ? "ACTUALPASSWORD" : "";
```

Comment:
Even though it's just a test implementation, let's return null instead of "" as the defaulting will happen at the ExpressionResolver instead.